plugins/widget/plugin.js

```diff
@@ -66,6 +62,16 @@
 				'.cke_widget_drag_handler_container:hover{' +
 					'opacity:1' +
 				'}' +
+				'.cke_widget_drag_handler_container_hidden{' +
```

Comment:
What is the purpose of introducing such class? It seems to be never applied to the widget's drag handler, causing funny bug when editor is loaded while mouse cursor is over place in which widget will appear:

<img width="189" alt="Image widget with drag-handler attached to the bottom" src="https://user-images.githubusercontent.com/1078728/52902362-e2a6ef00-320f-11e9-9780-bf47a2b95f8c.png">

Additionally I don't see a need to add such class. If `display: none` is a default style, it can be directly added to other styles of drag handler – no need of introducing another class.